components/scroller/scroller_default.story.vue

```diff
@@ -0,0 +1,313 @@
+    avatar: defaultAvatar,
+    message: messages[7],
+  },
+  {
+    id: 12,
+    avatar: defaultAvatar,
+    message: messages[2],
+  },
+  {
+    id: 13,
+    avatar: defaultAvatar,
+    message: messages[3],
+  },
+  {
+    id: 14,
+    avatar: defaultAvatar,
+    message: messages[3],
+  },
+  {
+    id: 15,
+    avatar: defaultAvatar,
+    message: messages[1],
+  },
+  {
+    id: 16,
+    avatar: defaultAvatar,
+    message: messages[0],
+  },
+  {
+    id: 17,
+    avatar: defaultAvatar,
+    message: messages[6],
+  },
+  {
+    id: 21,
+    avatar: defaultAvatar,
+    message: messages[7],
+  },
+  {
+    id: 22,
+    avatar: defaultAvatar,
+    message: messages[4],
+  },
+  {
+    id: 23,
+    avatar: defaultAvatar,
+    message: messages[3],
+  },
+  {
+    id: 24,
+    avatar: defaultAvatar,
+    message: messages[2],
+  },
+  {
+    id: 25,
+    avatar: defaultAvatar,
+    message: messages[6],
+  },
+  {
+    id: 26,
+    avatar: defaultAvatar,
+    message: messages[5],
+  },
+  {
+    id: 27,
+    avatar: defaultAvatar,
+    message: messages[3],
+  },
+  {
+    id: 28,
+    avatar: defaultAvatar,
+    message: messages[7],
+  },
+  {
+    id: 29,
+    avatar: defaultAvatar,
+    message: messages[0],
+  },
+  {
+    id: 210,
+    avatar: defaultAvatar,
+    message: messages[4],
+  },
+  {
+    id: 211,
+    avatar: defaultAvatar,
+    message: messages[4],
+  },
+  {
+    id: 212,
+    avatar: defaultAvatar,
+    message: messages[5],
+  },
+  {
+    id: 213,
+    avatar: defaultAvatar,
+    message: messages[6],
+  },
+  {
+    id: 214,
+    avatar: defaultAvatar,
+    message: messages[7],
+  },
+  {
+    id: 215,
+    avatar: defaultAvatar,
+    message: messages[0],
+  },
+  {
+    id: 216,
+    avatar: defaultAvatar,
+    message: messages[1],
+  },
+  {
+    id: 217,
+    avatar: defaultAvatar,
+    message: messages[2],
+  },
+]);
+
+const scroller = ref('scroller');
+
+function scrollToItem (index) {
+  scroller.value.scrollToItem(index);
+}
+
+function updateItem (index) {
+  dynamicItems.value[index].message = messages[Math.floor(Math.random() * 8)];
+}
+</script>
+
+<style scoped>
+.scroller {
+  height: 200px;
+  width: 100%;
+  max-width: calc(var(--size-300) * 157);
+  margin: 20px 0;
+  border: 1px solid red;
+}
+
+.user {
+  height: 25px;
+  padding: 0 12px;
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid #eee;
+}
+
+.image{
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  margin-right: 10px;
+  object-fit: cover;
+}
+</style>
+
+<style>
+.resize-observer[data-v-b329ee4c]{
+  position:absolute;
+  top:0;
+  left:0;
+  z-index:-1;
+  width:100%;
+  height:100%;
+  border:none;
+  background-color:transparent;
+  pointer-events:none;
+  display:block;
+  overflow:hidden;
+  opacity:0
+}
+.resize-observer[data-v-b329ee4c] object{
+   display:block;
+  position:absolute;
+  top:0;
+  left:0;
+  height:100%;
+  width:100%;
+  overflow:hidden;
+  pointer-events:none;
+  z-index:-1
+ }
```

Comment:
Ah yes, this is a leftover.
I used it because it was on the CSS of the library... it was here because the vue-resize package generate a element with 'resize-observer' and this data `data-v-b329ee4c`... so in the library they just hide it.

You can check it here https://jsfiddle.net/Akryum/zqygwf3r/

Reply:
I think the data-v-b329ee4c may change whenever they release a new build. Does it break anything if we remove this code?

Reply:
It not because I removed resize functionality and we are not using 'resize-observer' anymore

Reply:
great, we can remove it then.